### ISSUE TITLE
Update on F-Droid ?

### ISSUE THREAD

**Issue by IPv777** [TeamAmaze/AmazeFileManager#1886]
Hello,

the last version on F-Droid is a build of 2017, could you update it ?

Thanks,
IPv777

**Comment by TranceLove** [comment]
Blocked by #1593. Team is homebrewing our own library to replace it.

Duplicate of #1684.
